oteapi/models/filterconfig.py

```diff
@@ -11,13 +11,22 @@ class FilterConfig(GenericConfig):
     """Filter Strategy Data Configuration."""
 
     filterType: str = Field(
-        ..., description="Type of registered filter strategy. E.g., `filter/sql`."
+        ...,
+        description="Type of registered filter strategy. E.g., `filter/sql`.",
+        IRI="http://purl.org/dc/terms/type",  # type: ignore
+    )
+    query: Optional[str] = Field(
+        None,
+        description="Define a query operation.",
+        IRI="http://www.w3.org/1999/02/22-rdf-syntax-ns#Statement",  # type: ignore
```

Comment:
There exists no `rdfs:Statement`. However, there exists a `rdf:Statement`, but is not the correct property to use here. 

I haven't looked into the details of the [data cube](https://www.w3.org/TR/vocab-data-cube/) vocabulary, but  [qb:slice](https://www.w3.org/TR/vocab-data-cube/#ref_qb_slice_LC) seems to be a possible property one could refer to. Slicing is about selecting a subpart of a dataset, witch is exactly the purpose of a filter strategy.